packages/ui/src/components/va-button/VaButton.vue

```diff
@@ -250,82 +326,6 @@ export default defineComponent({
     }
   }
 
-  &--default {
-    color: var(--va-button-background-color, var(--va-white));
-
-    i {
-      color: var(--va-button-icon-color, var(--va-white));
-    }
-  }
-
-  &--outline {
-    background-color: transparent;
-    border: solid var(--va-button-outline-border, var(--va-outline-border-width));
-    text-decoration: none;
-
-    .va-button__content {
-      margin: calc(var(--va-button-outline-border, var(--va-outline-border-width)) * -1);
-    }
-
-    &.va-button--disabled {
-      background: transparent;
-
-      @include va-disabled;
-
-      &.va-button--active {
-        .va-button__content,
-        i {
-          color: var(--va-button-outline-icon-color, var(--va-white)) !important;
-        }
-      }
-    }
-  }
-
-  &--flat {
-    background: transparent;
-    border: var(--va-button-flat-border, var(--va-control-border)) solid transparent;
-    text-decoration: none;
-  }
-
-  &.va-button--disabled {
-    @include va-disabled;
-  }
-
-  &--large {
-    @include va-button(var(--va-button-lg-content-py), var(--va-button-lg-content-px), var(--va-button-lg-font-size), var(--va-button-lg-line-height), var(--va-button-lg-border-radius));
-
-    letter-spacing: var(--va-button-lg-letter-spacing);
-    min-height: var(--va-button-lg-size);
-    min-width: var(--va-button-lg-size);
-
-    .va-button__content {
-      padding: var(--va-button-lg-content-py) var(--va-button-lg-content-px);
-    }
-
-    &.va-button--round {
-      width: var(--va-button-lg-size);
-      height: var(--va-button-lg-size);
-    }
-
-    &.va-button--outline {
-      line-height: calc(var(--va-button-lg-line-height) - 2 * var(--va-button-outline-border, var(--va-outline-border-width)));
-    }
-
-    &.va-button--square {
-      border-radius: var(--va-button-lg-square-border-radius);
-    }
-
-    .va-button__left-icon {
-      margin-left: calc(var(--va-button-lg-content-px) / -2);
-      margin-right: calc(var(--va-button-lg-space-between-items) / 2);
-    }
-
-    .va-button__right-icon {
-      margin-left: calc(var(--va-button-lg-space-between-items) / 2);
-      margin-right: calc(var(--va-button-lg-content-px) / -2);
-    }
-  }
-
   &--small {
```

Comment:
How about big one? :P 

I think having solution based on sizes-config is more appropriate here if it is not actually a "dense" modifier.